legateboost/models/tree.py

```diff
@@ -46,13 +54,27 @@ def __init__(
         self,
         max_depth: int = 8,
         split_samples: int = 256,
-        alpha: float = 1.0,
+        l1_regularization: float = 0.0,
+        l2_regularization: float = 1.0,
```

Comment:
It is nice to have the actual constant here.  But is it normal that if you use `l1` regularization you *also* use `l2` regularization?

Because if not, it seems better to me to default to `None` on both set `l2_regularization = 1.0` only if both l1 and l2 are `None`.

Otherwise it seems an easy mistake to pass `l1_regularization=1.0` and not realize that l2 is also set.

Reply:
OK, thanks.  Still find this a bit surprising, but I now agree this seems to be what XGBoost does, so it's fine.